src/Reflection/SignatureMap/FunctionSignatureMapProvider.php

```diff
@@ -184,80 +184,54 @@ public function getSignatureMap(): array
 		$signatureMap = array_change_key_case($signatureMap, CASE_LOWER);
 
 		if ($this->stricterFunctionMap) {
-			$stricterFunctionMap = require __DIR__ . '/../../../resources/functionMap_bleedingEdge.php';
-			if (!is_array($stricterFunctionMap)) {
-				throw new ShouldNotHappenException('Signature map could not be loaded.');
-			}
-
-			$signatureMap = $this->computeSignatureMap($signatureMap, $stricterFunctionMap);
+			$signatureMap = $this->computeSignatureMapFile($signatureMap, 'functionMap_bleedingEdge.php');
```

Comment:
This makes the file not-clickable in the IDE which is worse experience.

Reply:
I didn't think about that. This can be easily fixed, see follow up